### ISSUE TITLE
Slope of ExponentialSmoothing prediction with dampening exceeds undamped slope

### ISSUE THREAD

**Issue by RUrlus** [statsmodels/statsmodels#6137]
#### Describe the bug

ExponentialSmoothing with `damped=True` yields higher, steeper sloped, forecasts than with `damped=False` for a positively trending time series.

This change was detected in our test set after updating to release 0.10.1. This issue is also present in the current master branch and not present in 0.10.0.
I believe this behaviour was introduced in commit 9f9c9c613705f27c283a192841deed0cacd001cf which changed the setting of the `initial_slope` by dividing it with `phi` (damping parameter).

As phi is expected to be bounded in (0, 1] and usually in [0.8, 0.98] dividing by phi increases the `initial_slope` parameter which I expect is the underlying cause of the bug.
At the moment I am not sure whether the appropriate step would be to multiply the `initial_slope` by `phi` when damped or not, but the current behaviour is not correct.

On a side note, why isn't `phi` set to 1 rather than to `np.nan` when damped is false? Mysteriously dividing `initial_slope` by NaN doesn't seem to raise an exception whereas multiplication by NaN does.

#### Code Sample, a copy-pastable example if possible

```python
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
from statsmodels.tsa.holtwinters import ExponentialSmoothing

np.random.seed(30)

mock_ts = pd.DataFrame(
    data=(
        np.linspace(1, 3, 1500) +
        np.cumsum(np.random.normal(0, 0.01, size=(1500)))
    ),
    index=np.arange(
        np.datetime64('1982-06-10 09:00:00', 'm'),
        np.datetime64('1982-06-11 10:00:00', 'm'),
        np.timedelta64(1, 'm')
    ),
    columns=['values']
)

# statsmodels wants a set frequency
mock_ts.index.freq = mock_ts.index.inferred_freq
train, test = mock_ts.iloc[:-120], mock_ts[-120:]

holt_pred = (
    ExponentialSmoothing(train, trend='mul', damped=False)
    .fit()
    .predict(test.index[0], test.index[-1])
    .values
)

damp_holt_pred = (
    ExponentialSmoothing(train, trend='mul', damped=True)
    .fit()
    .predict(test.index[0], test.index[-1])
    .values
)

fig = plt.figure(figsize=(14, 7))
plt.plot(holt_pred, label='Holt')
plt.plot(damp_holt_pred, label='Damped Holt')
plt.legend()
plt.show(fig)
# assert we project a function with a more conservative slope
# compared to holt
assert np.diff(damp_holt_pred).sum() <= np.diff(holt_pred).sum()
```
<details>

**Note**: As you can see, there are many issues on our GitHub tracker, so it is very possible that your issue has been posted before. Please check first before submitting so that we do not have to handle and close duplicates.

**Note**: Please be sure you are using the latest released version of `statsmodels`, or a recent build of `master`. If your problem has been fixed in an unreleased version, you might be able to use `master` until a new release occurs. 

**Note**: If you are using a released version, have you verified that the bug exists in the master branch of this repository? It helps the limited resources if we know problems exist in the current master so that they do not need to check whether the code sample produces a bug in the next release.

</details>

#### Expected Output

The expected result would be that the slope of the damped forecasts is smaller than that of the undamped forecast. This expected behavior was present up and until commit 9f9c9c613705f27c283a192841deed0cacd001cf.

#### Output of ``import statsmodels.api as sm; sm.show_versions()``

<details>

INSTALLED VERSIONS
------------------
Python: 3.7.3.final.0
OS: Darwin 18.7.0 Darwin Kernel Version 18.7.0: Thu Jun 20 18:42:21 PDT 2019; root:xnu-4903.270.47~4/RELEASE_X86_64 x86_64
byteorder: little
LC_ALL: en_US.UTF-8
LANG: None

Statsmodels
===========

Installed: 0.10.1 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/statsmodels)

Required Dependencies
=====================

cython: 0.29.13 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/Cython)
numpy: 1.17.0 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/numpy)
scipy: 1.3.0 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/scipy)
pandas: 0.25.0 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/pandas)
    dateutil: 2.8.0 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/dateutil)
patsy: 0.5.1 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/patsy)

Optional Dependencies
=====================

matplotlib: 3.1.1 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/matplotlib)
    backend: MacOSX 
cvxopt: Not installed
joblib: Not installed

Developer Tools
================

IPython: 7.7.0 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/IPython)
    jinja2: 2.10.1 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/jinja2)
sphinx: 2.2.0 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/sphinx)
    pygments: 2.4.2 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages/pygments)
pytest: 5.0.1 (/Users/rurlus/.pyenv/versions/3.7.3/envs/TOOLS/lib/python3.7/site-packages)
virtualenv: Not installed

</details>

**Comment by Rish001** [comment]
Hi can I work on this?

**Comment by bashtage** [comment]
Of course.

**Comment by bashtage** [comment]
Fixed in #6870
